logtape/formatter.ts

```diff
@@ -130,6 +131,8 @@ export interface TextFormatterOptions {
     | "time"
     | "date"
     | "rfc3339"
+    | "none"
+    | "disabled"
     | ((ts: number) => string);
```

Comment:
The custom callback also should be able to return `null`:

```suggestion
    | ((ts: number) => string | null);
```